plugins/extra/karma.go

```diff
@@ -60,25 +118,22 @@ func (p *karmaPlugin) cleanedName(name string) string {
 
 // GetKarmaFor returns the karma for the given name.
 func (p *karmaPlugin) GetKarmaFor(name string) int {
-	out := &KarmaTarget{Name: p.cleanedName(name)}
-
-	_ = p.db.View(func(tx *nut.Tx) error {
-		bucket := tx.Bucket("karma")
-		return bucket.Get(out.Name, out)
-	})
-
+	out := &Karma{Name: p.cleanedName(name)}
+	_, _ = p.db.Get(out)
```

Comment:
Why capture the return values?

Reply:
Using _ makes the linters happy... specifically because we're ignoring an error here.

Reply:
Would love a comment here then :)